src/hotspot/share/oops/symbolHandle.hpp

```diff
@@ -43,6 +65,10 @@
 // probe() and lookup_only() will increment the refcount if symbol is found.
 template <bool TEMP>
 class SymbolHandleBase : public StackObj {
+  static TempSymbolDelayQueue _cleanup_delay;
+  static volatile int32_t _cleanup_delay_len;
+  static volatile int32_t _cleanup_delay_max_entries;
```

Comment:
We don't usually use sized types unless the size actually matters.  I don't think it does here.